android/src/main/java/org/mozilla/mozstumbler/client/navdrawer/MainDrawerActivity.java

```diff
@@ -136,11 +136,15 @@ private void init() {
 
         getSupportActionBar().setTitle(getString(R.string.app_name));
 
+        getSupportActionBar().setHomeButtonEnabled(true);
```

Comment:
These 4 lines are needed for the icon logo to show up in the titlebar, but technically - this styling is in violation of the Material design guidelines.  We should do something else other than the icon for visual branding.